tests/contrib/psycopg/test_psycopg.py

```diff
@@ -291,6 +292,25 @@ def test_composed_query(self):
             dict(name='postgres.query', resource=query.as_string(db)),
         )
 
+    @snapshot()
```

Comment:
would have been great to have 2 different changes for this though, one for riot and one after for this test.

Reply:
Moved this into #2013 cc @brettlangdon